golang/UPDATING.md

```diff
@@ -18,4 +18,27 @@
 
 ## Publishing a new release
 
-Ask someone with write permission for the repo to follow the [instructions for publishing Go modules](https://go.dev/blog/publishing-go-modules).
\ No newline at end of file
+Publishing can only be performed by someone with commit privileges to the repo.
+
+1. Ensure you are the in branch (likely `master`) and at the commit you'd like to tag as a release version.
```

Comment:
typo?

Reply:
Ha I tried to fix this typo once already and got the order wrong :)